Add Isra University to the list

Eljinini opened the pull request:
Add Isra University to the list

philipto commented:
@Eljinini I am sorry, the domain iu.edu.jo is removed from this trusted domains list because it was abused in the past. If you are the student of this school, please enroll to [GitHub Student Developer Program](https://education.github.com/discount_requests/application). Then apply for JetBrains educational subscription with your GitHub account (**GitHub** tab at https://www.jetbrains.com/shop/eform/students).

Eljinini commented:
Dear Philip,

 �

Thank you for your response. I understand the reasoning behind the removal of the iu.edu.jo domain from the trusted list. However, I believe that generalizing based on past incidents may not accurately reflect the current situation.

As a result, I will explore alternative IDEs available in the market.

 �

Regards,

Mohammad Ali

 �


 �

"Mohammad Ali" H. Eljinini, PhD

Associate Professor, Department of Computer Information Systems 

Isra University, P.O. Box 22, 33 Isra Post Office, Amman, Jordan 11622

***@***.*** ***@***.***>   | Tel: 06 4711710, Ext: 2493

 

 <https://www.iu.edu.jo/>  <https://sites.google.com/view/eljinini>    <https://scholar.google.com/citations?user=HzrpnZgAAAAJ&hl=en>    <https://www.facebook.com/eljinini/>    <https://twitter.com/AliEljinini>   <https://www.youtube.com/channel/UCSU0bgzDupyuyJKupjpO9eQ>    <https://www.linkedin.com/in/mohammad-ali-eljinini/>    <https://www.researchgate.net/profile/Mohammad-Ali-Eljinini>    ***@***.***>   <https://www.scopus.com/authid/detail.uri?authorId=35738552000> 

 

 

From: Philip Torchinsky ***@***.***> 
Sent: Wednesday, February 12, 2025 4:48 PM
To: JetBrains/swot ***@***.***>
Cc: Mohammad Ali ***@***.***>; Mention ***@***.***>
Subject: Re: [JetBrains/swot] Add Isra University to the list (PR #27229)

 

@Eljinini <https://github.com/Eljinini>  I am sorry, the domain iu.edu.jo is removed from this trusted domains list because it was abused in the past. If you are the student of this school, please enroll to GitHub Student Developer Program <https://education.github.com/discount_requests/application> . Then apply for JetBrains educational subscription with your GitHub account (GitHub tab at https://www.jetbrains.com/shop/eform/students).

—
Reply to this email directly, view it on GitHub <https://github.com/JetBrains/swot/pull/27229#issuecomment-2653764267> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADSB4FP5RKW7XIOSAM6BYYL2PNGKVAVCNFSM6AAAAABW67J6R6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJTG43DIMRWG4> .
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/ADSB4FPOHVH6PQUI5WYCHW32PNGKVA5CNFSM6AAAAABW67J6R6WGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTU6FU5KW.gif> Message ID: ***@***.***>

philipto commented:
Dear @Eljinini,

thank you for your feedback. I re-reviewed the case of iu.edu.jo. In this case the violation was not severe, and it happened a while ago. I have a reason to believe that the university already implemented security measures protecting university emails from abuse.

Therefore, **I added the domain iu.edu.jo back to the trusted list.** 

Also please notice that students were always able to request their free licenses using GitHub accounts. It's quite usual way of getting student benefits worldwide, and it does not deprive the students from free licenses. 

Besides, I really hope that your choice of tools is based on their value for your work and your students, not on how many clicks is required to get a free license. Applying with GitHub account is very easy if a student has one, and takes just a little bit longer when they don't have it.

Pull request merged. **Please start requesting the licenses in about an hour, to let the changes to propagate through our system.** Thank you.

Eljinini commented:
Dear Philip Torchinsky,

 �

Thank you for taking the time to re-review the case and for reinstating iu.edu.jo to the trusted domain list. I truly appreciate your willingness to reconsider and acknowledge the security improvements made by the university.

I understand the importance of security in such programs and fully support the measures taken to prevent abuse. I also acknowledge the convenience of using GitHub accounts for student license applications and will ensure that students are aware of this process moving forward.

Again, I appreciate your prompt response and the support GitHub provides to educators and students. Thank you for your time and assistance.

 �

Best regards,
Mohammad Ali

 �


 �

"Mohammad Ali" H. Eljinini, PhD

Associate Professor, Department of Computer Information Systems 

Isra University, P.O. Box 22, 33 Isra Post Office, Amman, Jordan 11622

***@***.*** ***@***.***>   | Tel: 06 4711710, Ext: 2493

 

 <https://www.iu.edu.jo/>  <https://sites.google.com/view/eljinini>    <https://scholar.google.com/citations?user=HzrpnZgAAAAJ&hl=en>    <https://www.facebook.com/eljinini/>    <https://twitter.com/AliEljinini>   <https://www.youtube.com/channel/UCSU0bgzDupyuyJKupjpO9eQ>    <https://www.linkedin.com/in/mohammad-ali-eljinini/>    <https://www.researchgate.net/profile/Mohammad-Ali-Eljinini>    ***@***.***>   <https://www.scopus.com/authid/detail.uri?authorId=35738552000> 

 

 

 

 

From: Philip Torchinsky ***@***.***> 
Sent: Thursday, February 13, 2025 8:08 PM
To: JetBrains/swot ***@***.***>
Cc: Mohammad Ali ***@***.***>; Mention ***@***.***>
Subject: Re: [JetBrains/swot] Add Isra University to the list (PR #27229)

 

Dear @Eljinini <https://github.com/Eljinini> ,

thank you for your feedback. I re-reviewed the case of iu.edu.jo. In this case the violation was not severe, and it happened a while ago. I have a reason to believe that the university already implemented security measures protecting university emails from abuse.

Therefore, I added the domain iu.edu.jo back to the trusted list.

Also please notice that students were always able to request their free licenses using GitHub accounts. It's quite usual way of getting student benefits worldwide, and it does not deprive the students from free licenses.

Besides, I really hope that your choice of tools is based on their value for your work and your students, not on how many clicks is required to get a free license. Applying with GitHub account is very easy if a student has one, and takes just a little bit longer when they don't have it.

Pull request merged. Please start requesting the licenses in about an hour, to let the changes to propagate through our system. Thank you.

—
Reply to this email directly, view it on GitHub <https://github.com/JetBrains/swot/pull/27229#issuecomment-2657238087> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADSB4FMV3MPV3WHUUUXRC532PTGOLAVCNFSM6AAAAABW67J6R6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJXGIZTQMBYG4> .
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/ADSB4FJGI5YOPY557LGJHKD2PTGOLA5CNFSM6AAAAABW67J6R6WGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTU6MI6EO.gif> Message ID: ***@***.***>

  <https://avatars.githubusercontent.com/u/2843439?s=20&v=4> philipto left a comment (JetBrains/swot#27229) <https://github.com/JetBrains/swot/pull/27229#issuecomment-2657238087> 

Dear @Eljinini <https://github.com/Eljinini> ,

thank you for your feedback. I re-reviewed the case of iu.edu.jo. In this case the violation was not severe, and it happened a while ago. I have a reason to believe that the university already implemented security measures protecting university emails from abuse.

Therefore, I added the domain iu.edu.jo back to the trusted list.

Also please notice that students were always able to request their free licenses using GitHub accounts. It's quite usual way of getting student benefits worldwide, and it does not deprive the students from free licenses.

Besides, I really hope that your choice of tools is based on their value for your work and your students, not on how many clicks is required to get a free license. Applying with GitHub account is very easy if a student has one, and takes just a little bit longer when they don't have it.

Pull request merged. Please start requesting the licenses in about an hour, to let the changes to propagate through our system. Thank you.

—
Reply to this email directly, view it on GitHub <https://github.com/JetBrains/swot/pull/27229#issuecomment-2657238087> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADSB4FMV3MPV3WHUUUXRC532PTGOLAVCNFSM6AAAAABW67J6R6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNJXGIZTQMBYG4> .
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/ADSB4FJGI5YOPY557LGJHKD2PTGOLA5CNFSM6AAAAABW67J6R6WGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTU6MI6EO.gif> Message ID: ***@***.***>
